package.json

```diff
@@ -87,11 +87,49 @@
+    "test-unit-mssql":"DIALECT=mssql npm run test-unit",
+    "test-unit-all":"npm run test-unit-mysql && npm run test-unit-postgres && npm run test-unit-postgres-native && npm run test-unit-mssql && npm run test-unit-sqlite && npm run test-unit-mariadb",
+    "test-integration": "./node_modules/.bin/mocha --globals setImmediate,clearImmediate --ui tdd --check-leaks --colors -t 15000 --reporter spec --grep ''$GREP'' 'test/integration/**/*.test.js'",
+    "test-integration-mysql":"DIALECT=mysql npm run test-integration",
+    "test-integration-postgres":"DIALECT=postgres npm run test-integration",
+    "test-integration-postgres-native":"DIALECT=postgres-native npm run test-integration",
+    "test-integration-mariadb":"DIALECT=mariadb npm run test-integration",
+    "test-integration-sqlite":"DIALECT=sqlite npm run test-integration",
+    "test-integration-mssql":"DIALECT=mssql npm run test-integration",
+    "test-integration-all":"npm run test-integration-mysql && npm run test-integration-postgres && npm run test-integration-postgres-native && npm run test-integration-mssql && npm run test-integration-sqlite && npm run test-integration-mariadb",
+    "cover": "rm -rf coverage && npm run teaser && COVERAGE=true ./node_modules/.bin/istanbul cover ./node_modules/.bin/_mocha --report lcovonly -- -t 30000 --ui tdd 'test/integration/**/*.test.js'",
+    "mysql-cover": "DIALECT=mysql npm run cover && mv coverage coverage-mysql",
+    "sqlite-cover": "DIALECT=sqlite npm run cover && mv coverage coverage-sqlite",
+    "postgres-cover": "DIALECT=postgres npm run cover && mv coverage coverage-postgres",
+    "postgres-native-cover": "DIALECT=postgres-native npm run cover && mv coverage coverage-postgres-native",
+    "mariadb-cover": "DIALECT=mariadb npm run cover && mv coverage coverage-mariadb",
+    "mssql-cover": "DIALECT=mssql npm run cover && mv coverage coverage-mssql",
+    "merge-coverage": "rm -rf coverage && mkdir coverage && './node_modules/.bin/lcov-result-merger coverage-*/lcov.info' 'coverage/lcov.info'",
+    "codeclimate-send": "npm install -g codeclimate-test-reporter && CODECLIMATE_REPO_TOKEN=ce835a510bbf423a5ab5400a9bdcc2ec2d189d840b31657c6ee7cb9916b161d6 codeclimate < coverage/lcov.info",
+    "cover-all": "npm run mysql-cover && npm run postgres-cover && npm run postgres-native-cover && npm run mssql-cover && npm run sqlite-cover && npm run mariadb-cover && npm run merge-coverage",
+    "codeclimate": "npm run cover-all && npm run codeclimate-send",
+    "mysql": "DIALECT=mysql npm test",
+    "sqlite": "DIALECT=sqlite npm test",
+    "postgres": "DIALECT=postgres npm test",
+    "pgsql": "npm run postgres",
+    "postgres-native": "DIALECT=postgres-native npm test",
+    "postgresn": "npm run postgres-native",
+    "mariadb": "DIALECT=mariadb npm test",
+    "mssql": "DIALECT=mssql npm test",
+    "all": "npm run mysql && npm run sqlite && npm run postgres && npm run postgres-native && npm run mariadb && npm run mssql"
```

Comment:
imo we should kill these in favor of more descriptive `test-[dialect]` `test-unit-[dialect]` `test-integration[dialect]` `test-all`